app/scripts/modules/ecs/pipeline/stages/destroyAsg/ecsDestroyAsgStage.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+const angular = require('angular');
+
+import { PipelineTemplates, StageConstants } from '@spinnaker/core';
+
+module.exports = angular.module('spinnaker.ecs.pipeline.stage.ecs.destroyAsgStage', [])
+  .config(function(pipelineConfigProvider) {
+    pipelineConfigProvider.registerStage({
+      provides: 'destroyServerGroup',
+      alias: 'destroyAsg',
+      cloudProvider: 'ecs',
+      templateUrl: require('./destroyAsgStage.html'),
+      executionDetailsUrl: PipelineTemplates.destroyAsgExecutionDetails,
```

Comment:
`PipelineTemplates.destroyAsgExecutionDetails` no longer exists, you should be able to just omit the `executionDetailsUrl` property and the default execution details sections will be picked up from the default destroy asg stage.

Reply:
Sweet! I removed it just now